README.md

```diff
@@ -18,6 +18,13 @@ At Gitpod we've been asked a lot about how we do it. So we thought we might just
 
 ## Getting started
 
+### Gitpod
```

Comment:
I don’t think we should trying with sources in Gitpod in getting started. It is slow, since it is a  dev version and not something for end users. We have it already in development docs.

Reply:
Agree, feel free to take it out